### ISSUE TITLE
Ability to view/delete from SD card

### ISSUE THREAD

**Issue by Sonicboom247** [bambulab/BambuStudio#3261]
currently, the only way to view timelapse videos or photos from the SD card is to remove it and insert it into a PC. This is quite silly that the file structure is only one way.  I can constantly send files to the printer until the memory card is full. Then I need to remove the memory card to delete files manually. This is the polar opposite of what I wanted Bambu Studio and Bambu handy for.  It is simpler for me to go to my other brand of printer, and remove the thumb drive which does not need an adapter, delete or overwrite the old files, insert it into the printer and print. 

**Describe the solution you'd like**
I would like to either view and or delete files, and videos from the Bambu Handy or Bambu Studio.  either will work. or both.

**Describe alternatives you've considered**
Simply utilizing my old printer

**Additional context**

**Comment by masc2279** [comment]
This is not an issue. It right in front in the handy app and Bambu Studio.  

You can use any platform to view files, videos and images that are on the printers memroy card and delete as well as download not only on mobile but on all platforms using Bambu Studio. 

Its a basic feature and it is not hidden in any way in either the Bambu Studio or the Handy App.


<img width="1450" alt="Screenshot 2023-12-30 214241" src="https://github.com/bambulab/BambuStudio/assets/1273722/caea0dae-16a3-4902-b70b-958a23b1e4e3">


![IMG_2092](https://github.com/bambulab/BambuStudio/assets/1273722/df1e2654-2750-41be-b202-7b7539617305)

**Comment by djengis** [comment]
Wait, that is possible? I can do this with my P1P and my A1?

Always thought this is X1C only.

I get: Initialize failed (Not supported on the current printer version)!
Booth have the latest firmware. Booth the same message.

<img width="1710" alt="Bildschirmfoto 2023-12-31 um 13 29 48" src="https://github.com/bambulab/BambuStudio/assets/8014513/9d4b0052-6654-493e-b84d-4820c41c2248">

**Comment by Sonicboom247** [comment]
Thanks @djengis, that is exactly what i see.

**Comment by masc2279** [comment]
x

> Thanks @djengis, that is exactly what i see, hence the reason for the request

You did not fill out the report completly you did not mention anyting about version of device or software or anything that can help, other than a blanket ststement that clearly is wrong since it does work for some.

**Comment by Sonicboom247** [comment]
@masc2279, Nowhere on the request does it ask, mention, or indicate a device. My statement does indicate BOTH the Studio and the Handy app, Therefore, my statement is correct and your statement is in fact wrong since it does not work for some. You seem upset that your overuse of screenshots did not help.

**Comment by masc2279** [comment]
@Sonicboom247  Tell me what is this ?

<img width="1181" alt="Screenshot 2024-01-01 093102" src="https://github.com/bambulab/BambuStudio/assets/1273722/6010a5cf-b95a-4134-9577-921fd78a1677">

**Comment by masc2279** [comment]
@Sonicboom247   I am not mad there should be more information thats all. Your blanket statement _"currently, the only way to view timelapse videos or photos from the SD card is to remove it and insert it into a PC."_ just is not true it might be on specific devices but its not on all. I would have just passed thru if it clearly stated what device 

Same as 
_"Describe the solution you'd like
I would like to either view and or delete files, and videos from the Bambu Handy or Bambu Studio. either will work. or both."_

Well it does work for some devices.

Just saying add more informations to the write up to be fully clear what you are using and where the ask is.

**Comment by Sonicboom247** [comment]
Thanks for commenting but your link is to an old version that is not yet released for a different model printer. How does that make any sense? your link is for v1.8.0.57 with a P1, yet I am on 1.8.2.56 on an A1 with the latest firmware I can download and this option is still not there. so please, if anyone needs to do a quick search it's you.

**Comment by Sonicboom247** [comment]
Nope, sorry. Not confused in the least as there is nothing confusing about
firmware and software versions. Good try though. The sad part is that some
people are even toxic on here, perhaps instead of feeble attempts to attack
someone posting for help or to improve a product, only the pertinent
information is posted and save your emotions for your therapist.

**Comment by Sonicboom247** [comment]
Kkthxbai..lol

**Comment by wsnjie** [comment]
This is still an issue as of today with the A1 on firmware 1.01.01.00 and Bambu Studio 1.8.3.89

![Screenshot 2024-01-10 073326](https://github.com/bambulab/BambuStudio/assets/44382573/18772d86-4937-4b1d-962c-21f7e3665743)
![image_2024-01-10_073433076](https://github.com/bambulab/BambuStudio/assets/44382573/0377e0a4-efa0-4756-9025-99795a24a506)

**Comment by Sonicboom247** [comment]
The latest firmware from today has fixed this issue. I can now read and delete from the SD card
